Frontend/ui-common/src/Components/MobileNumberEditor/CountryCodeConfig.purs

```diff
@@ -0,0 +1,17 @@
+module Components.MobileNumberEditor.CountryCodeConfig where
+
+import Common.Types.App (CountryCodeObj)
+
+getCountryCodesObj :: Array CountryCodeObj
+getCountryCodesObj = [
+  {
+    countryName : "Bangladesh" 
+  , countryCode  : "+880" 
+  , countryShortCode : "BD"
+  },
+  {
+    countryName : "India" 
+  , countryCode  : "+91" 
+  , countryShortCode : "IN"
+  }
+]
```

Comment:
we can define config in controller only

Reply:
these object may grow large. that's why I put in separate.